### ISSUE TITLE
server crash when player join

### ISSUE THREAD

**Issue by keyblade** [swiftly-solution/swiftly#43]
Server start up is ok.When a player join, server crash.
If only with metamod, all fine.

startup cmd:
`cs2.exe -high -dedicated +map de_dust2 -maxplayers 64 +hostname "my server name" +game_mode 2 +sv_setsteamaccount "my server login token" +game_type 1 +sv_logfile 1`

metamod version:
`(dev build)  2.0 - build 1313`

swiftly version:
`1.2.14`

custom swiftly plugin(only one):
core.lua
```
function GetPluginAuthor()
    return "keyblade"
end

function GetPluginVersion()
    return "1.0.0"
end

function GetPluginName()
    return "fun_with_bots"
end

function GetPluginWebsite()
    return "xxx@yyy.com"
end
```

events.lua
```
AddEventHandler("OnPluginStart", function (event)
    logger:Write(4, "fun_with_bots start!")
end)
```

**Comment by skuzzis** [comment]
Hi there, what OS are you using?

**Comment by keyblade** [comment]
> Hi there, what OS are you using?

windows server 2012 R2 Datacenter

**Comment by skuzzis** [comment]
I've found the issue, being from the client language query.

https://github.com/swiftly-solution/swiftly/blob/149fddc18496c38586e103ad5a4bd04135f8db63/src/entrypoint.cpp#L514

We've temporarily removed the the Client Language feature from Windows until we investigate further. ([`a52d6128`](https://github.com/swiftly-solution/swiftly/commit/a52d6128f6ff1c92c115e01c78eb4084121bb112))

The assigned version for the above commit is `1.2.15`.

**Comment by keyblade** [comment]
> I've found the issue, being from the client language query.
> 
> https://github.com/swiftly-solution/swiftly/blob/149fddc18496c38586e103ad5a4bd04135f8db63/src/entrypoint.cpp#L514
> 
> We've temporarily removed the the Client Language feature from Windows until we investigate further. ([`a52d6128`](https://github.com/swiftly-solution/swiftly/commit/a52d6128f6ff1c92c115e01c78eb4084121bb112))
> 
> The assigned version for the above commit is `1.2.15`.

gj! wait for the release...

**Comment by blu133721** [comment]
[Released.](https://github.com/swiftly-solution/swiftly/releases/tag/v1.2.15)

**Comment by keyblade** [comment]
> [Released.](https://github.com/swiftly-solution/swiftly/releases/tag/v1.2.15)

still cause server crash...with this release i can see the team selection scene(can not select), and the server cmd windows disappeared suddenly at the same time.

**Comment by skuzzis** [comment]
Can you please install [AcceleratorCS2](https://github.com/Source2ZE/AcceleratorCS2) and send here the file created after the crash from `addons/AcceleratorCS2/dumps`? We need the `.dmp` file to debug the problem, because from our testings we couldn't reproduce it.

**Comment by keyblade** [comment]
> Can you please install [AcceleratorCS2](https://github.com/Source2ZE/AcceleratorCS2) and send here the file created after the crash from `addons/AcceleratorCS2/dumps`? We need the `.dmp` file to debug the problem, because from our testings we couldn't reproduce it.

[dumps.zip](https://github.com/user-attachments/files/17002942/dumps.zip)

**Comment by skuzzis** [comment]
I've removed the attachment due to it containing the steam token

**Comment by skuzzis** [comment]
I've pushed a fix in the commit [`1a9c7e50`](https://github.com/swiftly-solution/swiftly/commit/1a9c7e504d1f0e0a4d0fe19c690aa5b253589877).

The version assigned to this commit is `1.2.16`. This problem was a bit tricky due to the fact that we've couldn't retrieve any useful information from the dumps.

We're sorry for the problems caused due to the crash and we hope there will be no more in the future.
